### PR TITLE
br: public parameter `--ignore-stats` to backup statistic data

### PR DESCRIPTION
**Pull request by Leavrth**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
public parameter `--ignore-stats` to backup statistic data

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/48009

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by BornChanger — APPROVED** [review]

File: br/br-backup-statistics.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: 备份统计信息
```
Comment (benmaoer) [inline br/br-backup-statistics.md:2]: 建议合并文档到 br-snapshot-manual.md，方便用户看 BR 的手册查找到所有相关参数说明和场景说明

File: br/br-snapshot-manual.md [hunk 2]
```diff
@@ -109,6 +110,29 @@ br backup full \
     --log-file backupfull.log
 ```
 
+## 备份统计信息
+
+在默认情况下 BR 不会备份统计信息。但恢复数据后的集群可能会因为没有统计信息，无法选择最佳的执行计划。
+
+从 TiDB v7.5.0 起，备份恢复特性引入了备份统计信息的功能。该功能可以在备份数据的同时备份每张表的统计信息。
```
Comment (hfxsd) [inline br/br-snapshot-manual.md:117]:
```suggestion
从 TiDB v7.5.0 起，备份恢复特性引入了备份统计信息的功能。该功能可以在备份数据的同时备份每张表的统计信息。默认设置为 `ignore-stats=true`，即不备份统计信息。
```

File: br/br-snapshot-manual.md [hunk 3]
```diff
@@ -109,6 +110,29 @@ br backup full \
     --log-file backupfull.log
 ```
 
+## 备份统计信息
+
+在默认情况下 BR 不会备份统计信息。但恢复数据后的集群可能会因为没有统计信息，无法选择最佳的执行计划。
```
Comment (benmaoer) [inline br/br-snapshot-manual.md:115]:
```suggestion
从 TiDB v7.5.0 开始，br 命令行工具支持备份和恢复数据库的统计信息，包括列、索引和表级别的统计信息。通过支持该功能，从备份中恢复的 TiDB 数据库不再需要手动运行统计信息收集任务或等待自动收集任务的完成，简化了数据库维护并提升了查询性能。
```

File: br/br-snapshot-manual.md [hunk 4]
```diff
@@ -109,6 +110,29 @@ br backup full \
...
+
+从 TiDB v7.5.0 起，备份恢复特性引入了备份统计信息的功能。该功能可以在备份数据的同时备份每张表的统计信息。
+
+当你想要在备份集群数据的同时备份统计信息，你可以关闭参数 `--ignore-stats`：
```
Comment (benmaoer) [inline br/br-snapshot-manual.md:119]:
```suggestion
下面是备份集群快照数据并备份表统计信息的示例，需要设置 `--ignore-stats=false`：
```

File: br/br-snapshot-manual.md [hunk 5]
```diff
@@ -109,6 +110,29 @@ br backup full \
...
+
+```shell
+br backup full \
+--storage local:///br_data/ --pd "${PD_IP}:2379" --log-file restore.log \
+--ignore-stats=false
+```
+
+如果备份数据中存在统计信息，BR 在恢复数据时，也会自动恢复统计信息：
```
Comment (benmaoer) [inline br/br-snapshot-manual.md:127]:
```suggestion
在恢复数据时，如果备份集中存在统计信息，不需要额外的参数配置，br 命令行工具在恢复数据时自动恢复表统计信息：
```

File: br/br-snapshot-manual.md (hunk 2 above)
Comment (benmaoer) [inline br/br-snapshot-manual.md:117]:
```suggestion
br 命令行工具中新引入参数 `--ignore-stats` 控制备份数据时是否备份表统计信息。默认情况下，该参数是开启的，配置项为 `----ignore-stats=true`，也就是说，在备份数据时不备份表统计信息。
```

**Review by benmaoer — CHANGES_REQUESTED** [review]
修改部分描述

File: br/br-snapshot-manual.md [hunk 6]
```diff
@@ -109,6 +110,29 @@ br backup full \
     --log-file backupfull.log
 ```
 
+## 备份统计信息
+
+从 TiDB v7.5.0 开始，br 命令行工具支持备份和恢复数据库的统计信息，包括列、索引和表级别的统计信息。通过支持该功能，从备份中恢复的 TiDB 数据库不再需要手动运行统计信息收集任务或等待自动收集任务的完成，简化了数据库维护并提升了查询性能。
+
+从 TiDB v7.5.0 起，备份恢复特性引入了备份统计信息的功能。该功能可以在备份数据的同时备份每张表的统计信息。默认设置为 `ignore-stats=true`，即不备份统计信息。
```
Comment (hfxsd) [inline br/br-snapshot-manual.md:117]:
```suggestion
br 命令行工具中新引入参数 `--ignore-stats` 用于控制备份数据时是否备份表统计信息。该参数默认开启，配置项为 `--ignore-stats=true`，也就是说，在备份数据时不备份表统计信息。
```

File: br/br-snapshot-manual.md [hunk 7]
```diff
@@ -109,6 +110,29 @@ br backup full \
     --log-file backupfull.log
 ```
 
+## 备份统计信息
+
+从 TiDB v7.5.0 开始，br 命令行工具支持备份和恢复数据库的统计信息，包括列、索引和表级别的统计信息。通过支持该功能，从备份中恢复的 TiDB 数据库不再需要手动运行统计信息收集任务或等待自动收集任务的完成，简化了数据库维护并提升了查询性能。
+
+br 命令行工具中新引入参数 `--ignore-stats` 用于控制备份数据时是否备份表统计信息。该参数默认开启，配置项为 `--ignore-stats=true`，也就是说，在备份数据时不备份表统计信息。
```
Comment (hfxsd) [inline br/br-snapshot-manual.md:117]:
```suggestion
从 TiDB v7.5.0 开始，br 命令行工具引入参数 `--ignore-stats`，可以支持备份和恢复数据库的列、索引、和表级别的统计信息。启用该功能后，从备份中恢复的 TiDB 数据库不再需要手动运行统计信息收集任务，也无需等待自动收集任务的完成，从而简化了数据库维护工作，并提升了查询性能。

该参数默认开启，配置项为 `--ignore-stats=true`，即在备份数据时不备份表统计信息。
```

File: br/br-snapshot-manual.md [hunk 8]
```diff
@@ -109,6 +110,29 @@ br backup full \
...
+
+下面是备份集群快照数据并备份表统计信息的示例，需要设置 `--ignore-stats=false`：
+
+```shell
+br backup full \
+--storage local:///br_data/ --pd "${PD_IP}:2379" --log-file restore.log \
+--ignore-stats=false
+```
+
+在恢复数据时，如果备份集中包含了表统计信息，br 命令行工具默认在恢复数据时会自动恢复表统计信息：
```
Comment (hfxsd) [inline br/br-snapshot-manual.md:127]:
```suggestion
完成上述配置后，在恢复数据时，如果备份集中包含了表统计信息，br 命令行工具在恢复数据时会自动恢复表统计信息：
```

File: br/br-snapshot-manual.md [hunk 9]
```diff
@@ -109,6 +110,29 @@ br backup full \
...
```
Comment (hfxsd) [inline br/br-snapshot-manual.md:127]:
```suggestion
完成上述配置后，在恢复数据时，如果备份集中包含了表统计信息，br 命令行工具在恢复数据时会自动恢复表统计信息：
```

**Review by hfxsd — APPROVED** [review]

**Review by benmaoer — APPROVED** [review]
LGTM

File: br/br-snapshot-manual.md [hunk 10]
```diff
@@ -109,6 +110,29 @@ br backup full \
...
+
+下面是备份集群快照数据并备份表统计信息的示例，需要设置 `--ignore-stats=false`：
+
+```shell
+br backup full \
+--storage local:///br_data/ --pd "${PD_IP}:2379" --log-file restore.log \
+--ignore-stats=false
+```
+
+完成上述配置后，在恢复数据时，如果备份集中包含了表统计信息，br 命令行工具在恢复数据时会自动恢复表统计信息：
```
Comment (qiancai) [inline br/br-snapshot-manual.md:127]:
```suggestion
完成上述配置后，在恢复数据时，如果备份中包含了统计信息，br 命令行工具在恢复数据时会自动恢复备份的统计信息：
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**